.travis.yml

```diff
@@ -18,7 +19,10 @@ branches:
     - website
 
 script:
-  - npm run test && ./sig/import.sh && npm run make:darwin && node ./sig/publish.js
+  - npm run test
+  - npm run build
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]] && [[ $TRAVIS_PULL_REQUEST == "false" ]]; then ./sig/import.sh && npm run make:darwin && node ./sig/publish-mac.js; fi
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then npm run make:linux && echo Success; fi #&& node ./sig/publish-linux.js; fi
```

Comment:
Which line are you referring to? The normal build line doesn't invoke any of the `make:XXXX` rules, and the Linux line doesn't need it since it doesn't use any signing whatsoever.